portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -11,72 +11,71 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
 package com.google.sps.servlets;
 
-import java.io.IOException;
+import static java.util.Objects.isNull;
+
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.*; 
```

Comment:
nit: please import what you need explicitly. See https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Oops, I wasn't clear. Please remove `import java.util.*;` and explicitly import what you need.